tests/endpoints/internal.py

```diff
@@ -1,5 +1,361 @@
-        assert response.json() == {'status': 'OK'}
+    def test_healthcheck(self, mocked_config, client):
+        result = client.get(f'{mocked_config.internal_prefix}/healthcheck')
+
+        assert result.status_code == 200
+        assert result.json() == {'status': 'OK'}
+
+
+class TestInternalEndpoints:
+    payload_response_validator = {
+        'status_code': http.HTTPStatus.OK,
+        'json_schema': {
+            '$schema': 'https://json-schema.org/draft/2020-12/schema',
+            'properties': {'book_id': {'type': 'number'}, 'book_name': {'type': 'string'}}
+        }
+    }
+
+    payload_error_response = {'status_code': http.HTTPStatus.FORBIDDEN, 'body': {'detail': 'It is forbidden.'}}
+
+    expected_routes_response = [{
+        'hits': 0,
+        'response_selector': 'first',
+        'http_methods': ['GET'],
+        'path': '/users',
+        'response_validators': [
+            {
+                'status_code': 200,
+                'json_schema': {
+                    '$schema': 'https://json-schema.org/draft/2020-12/schema',
+                    'required': ['user_id'],
+                    'properties': {'user_id': {'type': 'number'}, 'user_name': {'type': 'string'}}
+                }
+            }
+        ],
+        'responses': [
+            {
+                'body': {'user_id': 1234, 'user_name': 'Mark Twain'},
+                'delay': [0.0, 0.0],
+                'headers': {},
+                'hits': 0,
+                'status_code': 200,
+                'weight': 1.0
+            }, {
+                'body': {'user_id': 5678, 'user_name': 'Charles Dickens'},
+                'delay': [0.0, 0.0],
+                'headers': {},
+                'hits': 0,
+                'status_code': 200,
+                'weight': 1.0
+            }
+        ]
+    }]
+
+    def test_get_routes(self, mocked_config, mocked_router, client):
+        expected_routes = copy.deepcopy(self.expected_routes_response)
+        expected_routes[0]['id'] = str(mocked_router.routes[0].id)
+        expected_routes[0]['response_validators'][0]['id'] = str(mocked_router.routes[0].response_validators[0].id)
+        expected_routes[0]['responses'][0]['id'] = str(mocked_router.routes[0].responses[0].id)
+        expected_routes[0]['responses'][1]['id'] = str(mocked_router.routes[0].responses[1].id)
+
+        result = client.get(f'{mocked_config.internal_prefix}/routes')
+
+        assert result.status_code == 200
+
+        assert result.json() == expected_routes
+
+    def test_get_route(self, mocked_config, mocked_router, client):
+        route_id = mocked_router.routes[0].id
+
+        result = client.get(f'{mocked_config.internal_prefix}/routes/{route_id}')
+
+        assert result.status_code == 200
+        assert result.json()['id'] == str(mocked_router.routes[0].id)
+
+        non_existent_id = uuid.uuid4()
+        result = client.get(f'{mocked_config.internal_prefix}/routes/{non_existent_id}')
```

Comment:
I'd rather split it to the another test. In general I'd recommend to keep one business rule per test.